### PR TITLE
migration: correct postcopy operation in migrate_gluster

### PR DESCRIPTION
**Pull request by Yingshun**
It did not set postcopy in previous code, update to
run virsh.migrate_postcopy for postcopy migration.

BTW, also update to use common checks in libvirt.MigrationTest.

Signed-off-by: Yingshun Cui <yicui@redhat.com>

### CODE REVIEW

**Comment by Yingshun** [comment]
**Test result:**
```
(.libvirt-ci-venv-ci-runtest-7ctyz5) [root@yicui-host1 ~]# avocado run --vt-type libvirt virsh.migrate_gluster
JOB ID     : 126ac95b8e2791f8edcd5e2ecc6195e4cc154c71
JOB LOG    : /root/avocado/job-results/job-2020-04-24T04.41-126ac95/job.log
 (1/4) type_specific.io-github-autotest-libvirt.virsh.migrate_gluster.backend_disk.mount_dir.with_postcopy: PASS (182.43 s)
 (2/4) type_specific.io-github-autotest-libvirt.virsh.migrate_gluster.backend_disk.mount_dir.without_postcopy: PASS (187.90 s)
 (3/4) type_specific.io-github-autotest-libvirt.virsh.migrate_gluster.backend_disk.mount_dir_softlink.with_postcopy: PASS (179.70 s)
 (4/4) type_specific.io-github-autotest-libvirt.virsh.migrate_gluster.backend_disk.mount_dir_softlink.without_postcopy: PASS (180.14 s)
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 732.49 s
(.libvirt-ci-venv-ci-runtest-7ctyz5) [root@yicui-host1 ~]# 
```

_output of set postcopy:_


```
2020-04-24 04:43:55,945 process          L0705 INFO | Command /bin/virsh -c 'qemu:///system' migrate --live --verbose --domain avocado-vt-vm1 --desturi qemu+ssh://10.66.85.245/system  --postcopy running on a thread
2020-04-24 04:43:56,205 virsh            L1471 WARNI| VM avocado-vt-vm1 does not exist
2020-04-24 04:43:57,836 libvirt          L1446 INFO | Migration started for avocado-vt-vm1
2020-04-24 04:43:57,836 virsh            L0655 DEBUG| Running virsh command: migrate-postcopy avocado-vt-vm1
2020-04-24 04:43:57,836 process          L0626 INFO | Running '/bin/virsh migrate-postcopy avocado-vt-vm1'
2020-04-24 04:43:57,853 process          L0458 DEBUG| [stdout]
2020-04-24 04:43:57,854 process          L0714 INFO | Command '/bin/virsh migrate-postcopy avocado-vt-vm1' finished with 0 after 0.014647245407104492s
2020-04-24 04:43:57,854 virsh            L0706 DEBUG| status: 0
2020-04-24 04:43:57,855 virsh            L0707 DEBUG| stdout:
```

**Comment by Yingshun** [comment]
depends on https://github.com/avocado-framework/avocado-vt/pull/1783

**Comment by chunfuwen** [comment]
Why the change depends on 1783 ? which is developed on year 2018 and still not integrated.

File: libvirt/tests/src/migration/migrate_gluster.py [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import logging
 import os
-import re
-
 
 from virttest import libvirt_vm
+from virttest import migration
```
Comment (chunfuwen) [inline libvirt/tests/src/migration/migrate_gluster.py:5]: you can not depend on https://github.com/avocado-framework/avocado-vt/pull/1783 ,which was developed on year 2018, and still not integrated.

**Comment by Yingshun** [comment]
1783 is updated recently so we decided to reuse it.

**Comment by Yingshun** [comment]
Test result:
(.libvirt-ci-venv-ci-runtest-7ctyz5) [root@yicui-host1 migration]# avocado run --vt-type libvirt virsh.migrate_gluster.backend_disk.mount_dir.with_postcopy
JOB ID     : 277db914d1c821d13c8d8ed530d292dfa59f18f5
JOB LOG    : /root/avocado/job-results/job-2020-04-27T02.49-277db91/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_gluster.backend_disk.mount_dir.with_postcopy: PASS (179.16 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 181.15 s
(.libvirt-ci-venv-ci-runtest-7ctyz5) [root@yicui-host1 migration]#

**Comment by kylazhang** [comment]
#1873 merged. So merge it. Thanks

**Pull request merged**